C/plugins/south/dht11/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 2.4.0)
+
+project(dht11)
+
+set(CMAKE_CXX_FLAGS_DEBUG "-O0 -ggdb")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+set(WIRINGPILIBS -lwiringPi -lwiringPiDev)
+
+# Find source files
+file(GLOB SOURCES *.cpp)
+
+# Include header files
+include_directories(include ../../../services/common/include ../common/include ../../../common/include ../../../thirdparty/rapidjson/include /usr/local/include/)
```

Comment:
do we need all of these? e.g. rapidjson

Reply:
Yes. RapidJson library would be required for parsing reconf request.